src/app/frontend/common/components/labels/labels_controller.js

```diff
@@ -19,6 +19,12 @@
 const alwaysVisibleLabelsNumber = 5;
 
 /**
+ * Regular expression matching URLs.
+ * @type {RegExp}
+ */
+const urlRegexp = new RegExp('(https?|ftp)://(-\.)?([^\s/?\.#-]+\.?)+(/[^\s]*)?', 'i');
```

Comment:
no i mean to change the regexp to '^(https?|ftp)://(-\.)?([^\s/?\.#-]+\.?)+(/[^\s]*)?$' because otherwise an annotation like "mixed-text-link: Foo http://k8s.io" would also match